lib/config/presets/internal/replacements.ts

```diff
@@ -120,4 +121,16 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'xmldom-to-scoped': {
+    description:
+      'the xmldom package is now published as @xmldom/xmldom, see https://github.com/xmldom/xmldom/issues/271',
```

Comment:
i don't think we need the issue link here
```suggestion
      'the xmldom package is now published as @xmldom/xmldom,',
```

Reply:
Just a question: Is this text only apearing in the documentaiton or also in the update PRs?

Reply:
only in the docs and renovate logs, and the cause for the package  rename doesn't matter for renovate.